gapis/capture/capture.proto

```diff
@@ -33,9 +33,11 @@ message File {
 // Record holds all the data for an entire capture.
 message Record {
   // Name of the capture.
-  string name = 1;
+  string key = 1;
```

Comment:
This is introduced because name is being used for other reasons then being a database key as well.

Reply:
The comment above is not matching the code anymore, can you edit the comment to:
`Key to index the capture`